mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -366,6 +366,9 @@ def process_code_coverage(key, value, timestamp):
                         self.logger.prn_inf("%s(%s)" % (key, str(value)))
                         result = value
                         event_queue.put(('__exit_event_queue', 0, time()))
+                    elif key == '__soft_reset_dut':
+                        # This event only resets the dut, not the host test
+                        dut_event_queue.put(('__reset_dut', True, time()))
```

Comment:
Let's make this key match the event queue key: `__reset_dut` -> `__soft_reset_dut`.